clusters/hive/pools/openshift-ci/ci-ocp-4-8-0-amd64-aws-us-east-1_clusterpool.yaml

```diff
@@ -22,15 +22,15 @@ spec:
   installAttemptsLimit: 1
   installConfigSecretTemplateRef:
     name: install-config-aws-us-east-1
-  maxSize: 20
+  maxSize: 5
```

Comment:
Dashboard shows this frequently hitting 20. I fear this is going to starve a bunch of jobs and people are going to come after you in slack...

But 4.9 looks like it generally stays below 13, so you could probably prune that one down some.

And it looks like the 4.10 us-west pool isn't using all its capacity, could probably get away with bumping that down to 13-15.

Reply:
Ideally we should have a controller to set this up automatically according to historical stats.
Currently, it is simple op that is shrink the oldest pool upon adding a pool.

I can leave 4.8 pool untouched (tune it when it cools down) but I do not want to resize 4.9 pool now because it might become more active again in the future.